dev/restinio/impl/parser_callbacks.ipp

```diff
@@ -243,10 +240,45 @@ restinio_message_complete_cb( http_parser * parser )
 	ctx->m_message_complete = true;
 	ctx->m_header.method( Http_Methods::from_nodejs( parser->method ) );
 
-	if( 0 == parser->upgrade )
-		ctx->m_header.should_keep_alive( 0 != http_should_keep_alive( parser ) );
+	if( 0 == llhttp_get_upgrade( parser ) )
+	{
+		ctx->m_header.should_keep_alive( 0 != llhttp_should_keep_alive( parser ) );
+	}
 	else
+	{
 		ctx->m_header.connection( http_connection_header_t::upgrade );
+	}
+
+	return HPE_PAUSED;
+}
+
+/*!
+ * @name Chunked encoding callbacks.
+ *
+ * @since v.0.7.0
+ */
+/// @{
+inline int
+restinio_chunk_extension_name_cb( llhttp_t * /*parser*/, const char * /*at*/, size_t /*length*/  )
```

Comment:
If these callbacks are left empty intentionally then it has to be noticed in comments.
Otherwise there should be marks `FIXME` or `TODO`.